<ui version="4.0" >
 <class>dEditField</class>
 <widget class="QMainWindow" name="dEditField" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>403</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>521</width>
    <height>343</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Field</string>
  </property>
  <widget class="QWidget" name="widget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Common</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="2" column="0" >
         <widget class="QLabel" name="tWidth" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Width</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QSpinBox" name="eWidth" >
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>254</number>
            </property>
            <property name="value" >
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tDec" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Decimals</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="eDec" />
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>230</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="textLabel4" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>32767</height>
           </size>
          </property>
          <property name="text" >
           <string>Type</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="textLabel1" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Name</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLineEdit" name="eName" />
        </item>
        <item row="3" column="0" colspan="3" >
         <widget class="Q3GroupBox" name="groupBox1" >
          <property name="title" >
           <string>Flags</string>
          </property>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QCheckBox" name="efSort" >
             <property name="text" >
              <string>Sort</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="efPlus" >
             <property name="text" >
              <string>Positive</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="efNZ" >
             <property name="text" >
              <string>Not null</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="efSum" >
             <property name="text" >
              <string>Sum</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QComboBox" name="eType" >
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="saldoTextLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Saldo source</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="3" >
         <widget class="QComboBox" name="combo_rests" />
        </item>
        <item row="8" column="0" >
         <widget class="QPushButton" name="saveField" >
          <property name="text" >
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" >
         <widget class="QPushButton" name="closeField" >
          <property name="text" >
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab1" >
       <attribute name="title" >
        <string>Description</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QTextEdit" name="eDescription" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabPage" >
       <attribute name="title" >
        <string>Aliases</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QTableWidget" name="tAliases" >
          <column>
           <property name="text" >
            <string>Lang</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <pixmapfunction>rcIcon</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >acfg.h</include>
  <include location="local" >mdtree.h</include>
  <include location="local" >qseditor.h</include>
  <include location="local" >aliaseditor.h</include>
  <include location="local" >mainform.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>eName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dEditField</receiver>
   <slot>nameChanged()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>165</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo_rests</sender>
   <signal>activated(int)</signal>
   <receiver>dEditField</receiver>
   <slot>AARegSelect(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>28</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveField</sender>
   <signal>clicked()</signal>
   <receiver>dEditField</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>51</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel" >
     <x>260</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeField</sender>
   <signal>clicked()</signal>
   <receiver>dEditField</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>128</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel" >
     <x>260</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
